### ISSUE TITLE
Comments for 2018/8/18 minutes

### ISSUE THREAD

**Issue by dc25** [dc25/seahug.github.io#1]
The comments for this issue will be used as comments for the seattlelinux.org 2018/08/18 meeting minutes: [https://dc25.github.io/seahug.github.io/minutes/2018/08/18/minutes.html](https://dc25.github.io/seahug.github.io/minutes/2018/08/18/minutes.html)

**Comment by dc25** [comment]
First comment

**Comment by dc25** [comment]
```haskell
--haskell syntax
main :: IO ()
main = do
  print "Hello World"
  print "Hello Again"
```

**Comment by dc25** [comment]
Some fancy formatting...
# big
## big
### big
#### big
##### big


- dog
- cat
- mouse


1. gold
2. silver
3. bronze

> Four score and seven years ago...

**Comment by dc25** [comment]
The sky is the limit...
![airplane](https://user-images.githubusercontent.com/1620/45274539-cc308680-b46c-11e8-88fa-f659d1c776c8.gif)

**Comment by rcook** [comment]
This is pretty nice! Thanks, Dave.

Here's some Haskell code from me:

```haskell
main :: IO ()
main = putStrLn "Hello, Dave!"
```

**Comment by rcook** [comment]
Yeah, I like the GitHub commenting approach. It's definitely more programmer-oriented which is a plus.
